MoviesGo/MoviesGo/DeteilModel/ViewModel/DetailsViewModel.swift

```diff
@@ -0,0 +1,88 @@
+// DetailsViewModel.swift
+// Copyright © Boris. All rights reserved.
+
+import Foundation
+
+protocol DetailsViewModelProtocol {
+    var details: DetailsMovie? { get set }
+    var updateData: VoidHendler? { get set }
+    var showError: ((String, Bool, @escaping VoidHendler) -> ())? { get set }
+    func loadData()
+}
+
+final class DetailsViewModel: DetailsViewModelProtocol {
+    // MARK: - Internal properties
+
+    var showError: ((String, Bool, @escaping VoidHendler) -> ())?
+    var details: DetailsMovie?
+    var updateData: VoidHendler?
+    var movieID: Int
+
+    // MARK: - Private propertie
+
+    private var networkLayer: MovieAPIServiceProtocol!
+
+    init(networkLayer: MovieAPIServiceProtocol, movieID: Int) {
+        self.networkLayer = networkLayer
+        self.movieID = movieID
+    }
+
+    private var stateView: ViewState<DetailsMovie> = .initial {
```

Comment:
```suggestion
    private var stateView: ViewState<DetailsMovie> = .initial {
```
это вынеси во вью или вообще удали но юзаешь не правильно!